crates/flipperzero/src/alloc.rs

```diff
@@ -0,0 +1,55 @@
+use core::alloc::{GlobalAlloc, Layout};
+use core::ffi::c_void;
+
+use flipperzero_sys::c_string;
+use flipperzero_sys::furi;
+
+extern "C" {
+    #[link_name = "free"]
+    pub fn free(p: *mut c_void);
+
+    #[link_name = "aligned_malloc"]
+    pub fn aligned_malloc(size: usize, align: usize) -> *mut *mut c_void;
```

Comment:
Why the double pointer to `void`? This doesn't match the definition in [`furi/core/memmgr.h`](https://github.com/flipperdevices/flipperzero-firmware/blob/0f9ea925d30f7a6183abe1002d680e3d1613a159/furi/core/memmgr.h#L45):

```c
void* aligned_malloc(size_t size, size_t alignment);
```